### ISSUE TITLE
Not working with tmdb helper 

### ISSUE THREAD

**Issue by tony2501** [Sam-Max/plugin.video.jacktook#72]
My configuration has debrid with torrentio, I am using the recommended .json file for the tmdb player configuration.  It doesn't work.  Any help?

**Comment by Sudiar31** [comment]
I do have same problem

**Comment by jollyhiggler** [comment]
Yes - it looks like it needs to be modified to conform with the new addon routing adjustments introduced in v0.4.0

If this functionality is essential to your setup you could always roll back to [v0.3.9](https://github.com/Sam-Max/plugin.video.jacktook/archive/35991fb5935355b48cb22ccdde90d7d8fe621ebb.zip) in the meantime.

**Comment by Sam-Max** [comment]
I will fix on next update, don't worry.

**Comment by Sam-Max** [comment]
Fixed on latest version 0.4.1

**Comment by Sudiar31** [comment]
Jacktook player works on the latest release.
👍👍👍

**Comment by daviddia5** [comment]
Hi, I can't find Jacktook player in tmdb helper. Do I need to do something special?

**Comment by Coffelius** [comment]
copy the file:
`addons/plugin.video.jacktook/jacktook.select.json` (you can also find it here https://github.com/Sam-Max/plugin.video.jacktook/blob/main/jacktook.select.json)

to the folder:
`userdata/addon_data/plugin.video.themoviedb.helper/players/`


Where userdata folder is in:

Operating system | Userdata Folder
-- | --
Android | Android/data/org.xbmc.kodi/files/.kodi/userdata/ (see note)
Flatpak | ~/.var/app/tv.kodi.Kodi/data/userdata
iOS | /private/var/mobile/Library/Preferences/Kodi/userdata/
LibreELEC | /storage/.kodi/userdata/
Linux | ~/.kodi/userdata/
macOS | /Users/<your_user_name>/Library/Application Support/Kodi/userdata/
Nvidia Shield (SMB) | smb://<nvidiashieldurl>/internal/Android/data/org.xbmc.kodi/files/.kodi/userdata
OSMC | /home/osmc/.kodi/userdata/
tvOS | /private/var/mobile/Library/Preferences/Kodi/userdata/
webOS | /media/developer/apps/usr/palm/applications/org.xbmc.kodi/.kodi/userdata/
Windows | %APPDATA%\Kodi\userdata
Windows Portable | <Install location chosen by you>\portable_data\userdata\
Windows via Microsoft Store | %LOCALAPPDATA%\Packages\XBMCFoundation.Kodi_4n2hpmxwrvr6p\LocalCache\Roaming\Kodi\
Windows Xbox | %LOCALAPPDATA%\Packages\XBMCFoundation.Kodi_4n2hpmxwrvr6p\LocalCache\Roaming\Kodi\

I don't know if there are other ways to do it

**Comment by daviddia5** [comment]
> copy the file: `addons/plugin.video.jacktook/jacktook.select.json` (you can also find it here https://github.com/Sam-Max/plugin.video.jacktook/blob/main/jacktook.select.json)
> 
> to the folder: `userdata/addon_data/plugin.video.themoviedb.helper/players/`
> 
> Where userdata folder is in:
> 
> Operating system	Userdata Folder
> Android	Android/data/org.xbmc.kodi/files/.kodi/userdata/ (see note)
> Flatpak	~/.var/app/tv.kodi.Kodi/data/userdata
> iOS	/private/var/mobile/Library/Preferences/Kodi/userdata/
> LibreELEC	/storage/.kodi/userdata/
> Linux	~/.kodi/userdata/
> macOS	/Users/<your_user_name>/Library/Application Support/Kodi/userdata/
> Nvidia Shield (SMB)	smb:///internal/Android/data/org.xbmc.kodi/files/.kodi/userdata
> OSMC	/home/osmc/.kodi/userdata/
> tvOS	/private/var/mobile/Library/Preferences/Kodi/userdata/
> webOS	/media/developer/apps/usr/palm/applications/org.xbmc.kodi/.kodi/userdata/
> Windows	%APPDATA%\Kodi\userdata
> Windows Portable	\portable_data\userdata\
> Windows via Microsoft Store	%LOCALAPPDATA%\Packages\XBMCFoundation.Kodi_4n2hpmxwrvr6p\LocalCache\Roaming\Kodi\
> Windows Xbox	%LOCALAPPDATA%\Packages\XBMCFoundation.Kodi_4n2hpmxwrvr6p\LocalCache\Roaming\Kodi\
> I don't know if there are other ways to do it

Thanks! The problem is that Android doesn't let you access system folders. But I finally got it through Kodi's file explorer.
